library/std/src/os/windows/process.rs

```diff
@@ -266,15 +268,17 @@ pub trait CommandExt: Sealed {
     /// #         let _ = self.0.kill();
     /// #     }
     /// # }
-    ///
+    /// #
     /// let parent = Command::new("cmd").spawn()?;
-    ///
     /// let mut child_cmd = Command::new("cmd");
     ///
     /// const PROC_THREAD_ATTRIBUTE_PARENT_PROCESS: usize = 0x00020000;
-    ///
     /// unsafe {
-    ///     child_cmd.raw_attribute(PROC_THREAD_ATTRIBUTE_PARENT_PROCESS, parent.as_raw_handle() as isize);
+    ///     child_cmd.raw_attribute(
+    ///         PROC_THREAD_ATTRIBUTE_PARENT_PROCESS,
+    ///         parent.as_raw_handle(),
```

Comment:
It will probably be necessary to change this to core::ptr::addr_of!(process_handle)

Reply:
Yes. In the old code, it takes the `value` argument, boxes it, and passes the box's data ptr as the `lpValue` argument to `UpdateProcThreadAttribute`. In the new code it directly passes `value_ptr` through to `UpdateProcThreadAttribute`. So an extra layer of reference needs to be added somewhere.